config/application.rb

```diff
@@ -21,7 +21,7 @@ class Application < Rails::Application
       'font-src https://cdn.jsdelivr.net',
       "img-src 'self' http: https:",
       "script-src 'self' https://cdnjs.cloudflare.com https://code.jquery.com https://maxcdn.bootstrapcdn.com",
-      "style-src 'self' https://maxcdn.bootstrapcdn.com"
+      "style-src 'self' 'unsafe-inline' https://maxcdn.bootstrapcdn.com"
```

Comment:
This is a temporary change to allow the use of inline style in "index.html.erb" (line 12) to fulfill the requirement that the images be displayed with a maximum width of 400px.  In a future change, CSS may be introduced, at which point we will revert the change to this line.

Reply:
This is fine, but it would also be okay to add a basic css file.